pipelines/src/main/scala/com/fulcrumgenomics/str/pipelines/PostDuplexConsensusPipeline.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+package com.fulcrumgenomics.str.pipelines
+
+import dagr.commons.io.{Io, PathUtil}
+import dagr.core.cmdline.Pipelines
+import dagr.core.tasksystem.Pipeline
+import dagr.sopt.{arg, clp}
+import dagr.tasks.DagrDef.{PathPrefix, PathToFasta, PathToVcf}
+import dagr.tasks.fgbio.{ClipOverlappingReads, FilterConsensusReads}
+import dagr.tasks.picard._
+import dagr.tasks.vc.VarDictJavaEndToEnd
+import htsjdk.variant.vcf.VCFFileReader
+
+
+@clp(
+  description =
+    """
+      |Pipeline to filter duplex consensus reads and compute metrics.
+      |
+      |Requires as input the output path prefix given to DuplexConsensusPipeline.
```

Comment:
As suggested, I think I would suggest moving the filtering step out of here so it can run on any BAM.  I would also suggest that it take in `input: PathToBam` and `intervals: PathToIntervals` instead of `input:PathPrefix`